packages/allow-scripts/README.md

```diff
@@ -2,20 +2,27 @@
 
 A tool for running only the dependency lifecycle hooks specified in an _allowlist_.
 
+> For an overview of LavaMoat tools see [the main README](https://github.com/LavaMoat/LavaMoat/tree/main/README.md)
+
+
 ### Install
 
 Adds the package to start using it in your project. be sure to include the `@lavamoat/` namespace in the package name
 
 ```sh
 yarn add -D @lavamoat/allow-scripts
+npm i -D @lavamoat/allow-scripts
 ```
 
 ### Setup
 
 ```sh
 yarn allow-scripts setup
+npx allow-scripts setup
 ```
 
+> **Warning** if @lavamoat/allow-scripts was not installed prior, npx will download and run allow-scripts (note no namespace prefix) which serves the same purpose but with different functionality and security gurantees.
```

Comment:
If there are different functionality and security guarantees, wouldn't it be better to recommend that people add `@lavamoat/allow-scripts` to their project? Unless you say that below, in which case — maybe this part should be removed?

Reply:
I meant it as a warning - don't use the other one accidentally, because it's less secure
Just don't want to :hankey: on a package this was based on ;) 

I'll work on this part, because the way you understood it is not what I intended.

Reply:
Check the new one